frontend/www/js/omegaup/components/problem/List.vue

```diff
@@ -6,14 +6,13 @@
       :initial-keyword="keyword"
       :tags="tags"
     ></omegaup-problem-search-bar>
-    <a
-      href="#"
-      class="d-inline-block mb-3"
-      role="button"
+    <button
+      class="btn btn-primary problem-finder-button"
```

Comment:
intentamos limitar el uso de clases personalizadas con fines de agregar estilos de CSS si ya existe una manera de expresar lo mismo. En este caso, se pueden usar las clases de [Bootstrap](https://getbootstrap.com/docs/4.5/utilities/spacing/). Y resulta que una de esas clases ya estaba incluida antes!

```suggestion
      class="btn btn-primary mb-3"
```

con eso debería ser suficiente!

Reply:
genial, gracias!